docker-compose.yaml

```diff
@@ -1,26 +1,58 @@
-version: '3.9.1'
+version: "3.9.1"
 
-# You must manually run `ankerctl config import` atleast once
-# docker run \
-# -v ankerctl_vol:/root/.config/ankerctl \
-# -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-# ankerctl config import /tmp/login.json
 services:
-  ankerctl:
-    image: ankerctl/ankerctl:latest
-    container_name: ankerctl
-    restart: unless-stopped
-    build: .
-    environment:
-      - FLASK_PORT=4470
-      - FLASK_HOST=0.0.0.0
-    volumes:
-      - ankerctl_vol:/root/.config/ankerctl
-    ports:
-      - 127.0.0.1:4470:4470
-    entrypoint: "/app/ankerctl.py"
-    command: ["webserver", "run"]
+    ankerctl_seed_mac:
+        image: busybox
+        container_name: ankerctl_seed_mac
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            - "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
+        command: sh -c "cp /tmp/login.json /root/.config/ankerctl/login.json || true"
+    ankerctl_seed_linux:
+        image: busybox
+        container_name: ankerctl_seed_linux
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            # TODO: verify this is the correct linux path
+            - "$HOME/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
```

Comment:
This would be our best bet, for Linux users running ankermake through wine with default settings:

```sh
$HOME/.wine/drive_c/users/$USER/AppData/Local/AnkerMake/AnkerMake_64bit_fp/login.json
```